src/main/java/com/azure/cosmos/examples/analyticalcontainercrud/async/AnalyticalContainerCRUDQuickstartAsync.java

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.examples.analyticalcontainercrud.async;
+
+import com.azure.cosmos.ConsistencyLevel;
+import com.azure.cosmos.CosmosAsyncClient;
+import com.azure.cosmos.CosmosAsyncContainer;
+import com.azure.cosmos.CosmosAsyncDatabase;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.examples.common.AccountSettings;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosContainerRequestOptions;
+import com.azure.cosmos.models.CosmosContainerResponse;
+import com.azure.cosmos.models.CosmosDatabaseRequestOptions;
+import com.azure.cosmos.models.CosmosDatabaseResponse;
+
+import reactor.core.publisher.Mono;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AnalyticalContainerCRUDQuickstartAsync {
+
+    private CosmosAsyncClient client;
+
+    private final String databaseName = "AzureSampleFamilyDB";
+    private final String containerName = "FamilyContainer";
+
+    private CosmosAsyncDatabase database;
+
+    protected static Logger logger = LoggerFactory.getLogger(AnalyticalContainerCRUDQuickstartAsync.class);
```

Comment:
I don't know why our samples have this as `protected`, though it doesn't matter much, but this should be 
```suggestion
private static final Logger logger = LoggerFactory.getLogger(AnalyticalContainerCRUDQuickstartAsync.class);
```

Reply:
@TheovanKraay  I think you missed this comment